src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md

```diff
@@ -13,12 +13,25 @@ Creates a VMSS.
 
 ## SYNTAX
 
+### DefaultParameter (Default)
 ```
 New-AzureRmVmss [-ResourceGroupName] <String> [-VMScaleSetName] <String>
  [-VirtualMachineScaleSet] <PSVirtualMachineScaleSet> [-AsJob] [-DefaultProfile <IAzureContextContainer>]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
+### SimpleParameterSet
+```
+New-AzureRmVmss [-AsJob] [[-ResourceGroupName] <String>] [-VMScaleSetName] <String> [-ImageName <String>]
```

Comment:
Not sure how the -AsJob ended up first in the order if it's last on the list of parameters.... can you regenerate one more time and see if it moves to the end?